src/components/AttachmentModal.js

```diff
@@ -115,7 +115,7 @@ class AttachmentModal extends PureComponent {
      * @param {String} sourceURL
      */
     downloadAttachment(sourceURL) {
-        fileDownload(sourceURL, this.props.originalFileName);
+        fileDownload(this.props.isAuthTokenRequired ? addEncryptedAuthTokenToURL(sourceURL) : sourceURL, this.props.originalFileName);
```

Comment:
Just noting that @kidroca add a comment about also being able to let `fileDownload` work with headers on the old PR: https://github.com/Expensify/App/pull/12648#discussion_r1032172931

Maybe we can add this in a follow-up PR?

Reply:
Yes I think a follow up PR is the right place for this so its not blocking the rest of this :) Thanks for keeping a note of that 🙌